src/test/java/tests/CasePaginationTests.java

```diff
@@ -95,4 +111,15 @@ public void testSearchAndPagination() throws Exception {
         assert entityListResponse.getPageCount() == 2;
         assert entityListResponse.getCurrentPage() == 1;
     }
+
+    public class FormPlayerPropertyManagerMock extends FormplayerPropertyManager {
+        public FormPlayerPropertyManagerMock(IStorageUtilityIndexed properties) {
+            super(properties);
+        }
+
+        @Override
+        public boolean isFuzzySearchEnabled() {
```

Comment:
can we add a setter for it and return that value instead instead of always returning `false`.